pkg/backend/openshiftcluster.go

```diff
@@ -235,6 +255,9 @@ func (ocb *openShiftClusterBackend) updateAsyncOperation(ctx context.Context, lo
 				asyncdoc.OpenShiftCluster = &ocCopy
 			}
 
+			clusterResult := newClusterResult(resultType, initialProvisioningState.String(), backendErr)
+			log.Print("Long running result: %v", *clusterResult)
```

Comment:
use %s and add String() to clusterResult

Reply:
Changed to `log.Printf("Long running result: %s", clusterResult.String())`